services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
+  try {
+    const user = await User.create(data);
+    return user;
+  } catch (error) {
+    throw error;
+  }
```

Comment:
This code snippet appears to be an empty block, which serves no purpose in the code. It should be removed to maintain clean and concise code.